<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>copyDetectDialog</class>
 <widget class="QDialog" name="copyDetectDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>225</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Copying - Device detection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="diskmasterui.qrc">
    <normaloff>:/DiskMasterUI/icons/copy_disk.png</normaloff>:/DiskMasterUI/icons/copy_disk.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Task name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="taskNameEdit_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Work folder:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLineEdit" name="workFolderEdit_"/>
         </item>
         <item>
          <widget class="QToolButton" name="openFolderButton_2">
           <property name="text">
            <string>...</string>
           </property>
           <property name="icon">
            <iconset resource="diskmasterui.qrc">
             <normaloff>:/DiskMasterUI/icons/folder_home.png</normaloff>:/DiskMasterUI/icons/folder_home.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Soruce device:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="sourceCombo_"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="diskmasterui.qrc">:/DiskMasterUI/icons/right_arrow.png</pixmap>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Target device:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="targetCombo_">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="fullCopyButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Full copy</string>
       </property>
       <property name="icon">
        <iconset resource="diskmasterui.qrc">
         <normaloff>:/DiskMasterUI/icons/hard_drive_clone.png</normaloff>:/DiskMasterUI/icons/hard_drive_clone.png</iconset>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="partitionCopyButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Partition copy</string>
       </property>
       <property name="icon">
        <iconset resource="diskmasterui.qrc">
         <normaloff>:/DiskMasterUI/icons/Partition.png</normaloff>:/DiskMasterUI/icons/Partition.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="userCopyButton">
       <property name="text">
        <string>Custom copy</string>
       </property>
       <property name="icon">
        <iconset resource="diskmasterui.qrc">
         <normaloff>:/DiskMasterUI/icons/user_copy.png</normaloff>:/DiskMasterUI/icons/user_copy.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string>Preferences</string>
       </property>
       <property name="icon">
        <iconset resource="diskmasterui.qrc">
         <normaloff>:/DiskMasterUI/icons/preferences.png</normaloff>:/DiskMasterUI/icons/preferences.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextUnderIcon</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder></zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>line_3</zorder>
 </widget>
 <resources>
  <include location="diskmasterui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>112</x>
     <y>130</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>341</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>copySettingClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>485</x>
     <y>144</y>
    </hint>
    <hint type="destinationlabel">
     <x>459</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fullCopyButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>fullCopyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>33</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>63</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>partitionCopyButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>partitionCopyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>198</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>169</x>
     <y>194</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>userCopyButton</sender>
   <signal>clicked()</signal>
   <receiver>copyDetectDialog</receiver>
   <slot>userCopyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>310</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>copySettingClicked()</slot>
  <slot>fullCopyClicked()</slot>
  <slot>partitionCopyClicked()</slot>
  <slot>userCopyClicked()</slot>
 </slots>
</ui>
